diff --git a/node_modules/@types/theme-ui__components/index.d.ts b/node_modules/@types/theme-ui__components/index.d.ts
index a5e8f4b..566cf3f 100644
--- a/node_modules/@types/theme-ui__components/index.d.ts
+++ b/node_modules/@types/theme-ui__components/index.d.ts
@@ -29,6 +29,11 @@ export interface BoxOwnProps extends SpaceProps, ColorProps {
     css?: InterpolationWithTheme<any>;
 }
 export interface BoxProps extends Assign<React.ComponentProps<'div'>, BoxOwnProps> {}
+
+export interface BoxProps extends Assign<React.ComponentProps<'a'>, BoxOwnProps> {
+    as?: 'a';
+    href?: string;
+}
 /**
  * Use the Box component as a layout primitive to add margin, padding, and colors to content.
  * @see https://theme-ui.com/components/box
