diff --git a/node_modules/@types/theme-ui/index.d.ts b/node_modules/@types/theme-ui/index.d.ts
index e3440ce..0a42f91 100644
--- a/node_modules/@types/theme-ui/index.d.ts
+++ b/node_modules/@types/theme-ui/index.d.ts
@@ -16,6 +16,7 @@ import * as CSS from 'csstype';
 import * as React from 'react';
 import { Theme as StyledSystemTheme } from 'styled-system';
 import { Interpolation, SerializedStyles } from '@emotion/serialize';
+import { Components } from '@mdx-js/react';
 export * from '@theme-ui/components';
 
 export {};
@@ -24,6 +25,7 @@ type ObjectOrArray<T> = T[] | { [K: string]: T | ObjectOrArray<T> };
 
 export interface ThemeProviderProps<Theme> {
     theme: Partial<Theme> | ((outerTheme: Theme) => Theme);
+    components?: Components
     children?: React.ReactNode;
 }
 
