diff --git a/node_modules/svelte/internal/index.mjs b/node_modules/svelte/internal/index.mjs
index ef86268..1235213 100644
--- a/node_modules/svelte/internal/index.mjs
+++ b/node_modules/svelte/internal/index.mjs
@@ -304,7 +304,7 @@ function insert(target, node, anchor) {
     }
 }
 function detach(node) {
-    node.parentNode.removeChild(node);
+    node.parentNode?.removeChild(node);
 }
 function destroy_each(iterations, detaching) {
     for (let i = 0; i < iterations.length; i += 1) {
